### PR TITLE
This is an example how to use Lightbend Telemetry (Cinnamon) for Log Correlation

### PR DESCRIPTION
**Pull request by ygree**

### CODE REVIEW

**Comment by ygree** [comment]
Since Cinnamon team has recently implemented this functionality, I've updated this PR to use this functionality and removed custom one.

File: lightbend-telemetry/log-correlation-java-mvn/.gitignore [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+.cache-tests
+.classpath
+.project
+.tmpBin
+.factorypath
+.settings
+logs
+.idea
+*.iml
+RUNNING_PID
```
Comment (ignasi35) [inline lightbend-telemetry/log-correlation-java-mvn/.gitignore]: do we need the per-folder `.gitiignore`?
These changes look valuable at root-level. I'm +1 to moving them to `/.gitignore`.

File: lightbend-telemetry/log-correlation-java-mvn/README.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
+# This is an example how to use Lightbend Telemetry (Cinnamon) for Log Correlation
+
+How it work:
```
Comment (ignasi35) [inline lightbend-telemetry/log-correlation-java-mvn/README.md]: `s/How it work/How it works/` (typo: missing `s`)

File: lightbend-telemetry/log-correlation-java-mvn/README.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
+Cinnamon starting 2.10.2 support auto log correlation generation.
```
Comment (ignasi35) [inline lightbend-telemetry/log-correlation-java-mvn/README.md]: I think we can link to actual docs too: https://developer.lightbend.com/docs/cinnamon/current//extensions/mdc.html to increase docs exposure.

File: lightbend-telemetry/log-correlation-java-mvn/README.md [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+``` 
+
+How to run:
+
+```
+mvn install                  # builds the service and downloads the Cinnamon agent
+mvn -pl hello-impl exec:exec # runs the hello service
+```
+
+Test:
```
Comment (ignasi35) [inline lightbend-telemetry/log-correlation-java-mvn/README.md]: It'd be great if we had a little explanation of what these logs and `curl` statement demonstrate. I had to go check the implementation in `HelloServiceImpl` to confirm that there was a remote call involved (the fact that both methods are in `HelloServiceImpl` and on the same process didn't help).

File: lightbend-telemetry/log-correlation-java-mvn/.gitignore (hunk 1 above)
Comment (ygree) [reply lightbend-telemetry/log-correlation-java-mvn/.gitignore]: Fixed

**Pull request merged**